mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -320,6 +320,7 @@ class LocalLDAModel private[spark] (
 
         docBound
       }.sum()
+    ElogbetaBc.destroy(blocking = false)
```

Comment:
@zhengruifeng but the broadcast isn't actually used. Its `.value` is called, locally, not from a distributed method.